src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import core.basesyntax.interfaces.AreaCalculator;
+import core.basesyntax.interfaces.Describer;
+
+public abstract class Figure implements Describer, AreaCalculator {
```

Comment:
Don't use verbs for class/interface names. The interface names 'Describer' and 'AreaCalculator' should be nouns or noun phrases, so consider renaming them to something like 'DescriptionProvider' and 'AreaProvider' if they provide descriptions and areas respectively.